Classes/Private/Wrapper/OCTToxAV.m

```diff
@@ -258,7 +388,16 @@ void callstateCallback(ToxAV *cToxAV,
                        uint32_t state,
                        void *userData)
 {
-    // To Do..
+    OCTToxAV *toxAV = (__bridge OCTToxAV *)userData;
+
+    dispatch_async(dispatch_get_main_queue(), ^{
+        DDLogCInfo(@"%@: callstateCallback from friend %d with state: %d", cToxAV, friendNumber, state);
+
+
+        if ([toxAV.delegate respondsToSelector:@selector(toxAV:callStateChanged:friendNumber:)]) {
+            [toxAV.delegate toxAV:toxAV callStateChanged:(OCTToxAVCallState)state friendNumber:friendNumber];
```

Comment:
Oh sorry, I wasn't accurate enough. What I meant was to create `switch` statement, in which match toxav state and OCTToxAVCallState